__tests__/Ship.test.js

```diff
@@ -0,0 +1,80 @@
+const Ship = require('../src/ship.js');
+const Port = require('../src/port.js');
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Ship', () => {
+  describe('with ports and an itinerary', () => {
+    let ship;
+    let dover;
+    let calais;
+    let itinerary;
+  
+    beforeEach(() => {
+      dover = new Port('Dover');
+      calais = new Port('Calais');
+      itinerary = new Itinerary([dover, calais]);
+      ship = new Ship(itinerary);
+    });
+   
+       it('has a starting port', () => {
+           const port = new Port('Dover');
+           const itinerary = new Itinerary([port]);
+           const ship = new Ship(itinerary);
+         
+           expect(ship.currentPort).toBe(port);
+         });
+         it('can set sail', () => {
```

Comment:
Its common convention to leave a clear line between each `it` block to help with readability, should add one in here